model/task_lifecycle.go

```diff
@@ -293,21 +293,17 @@ func TryResetTask(ctx context.Context, settings *evergreen.Settings, taskId, use
 
 	maxExecution := evergreen.MaxTaskExecution
 
-	if evergreen.IsCommitQueueRequester(t.Requester) && evergreen.IsSystemFailedTaskStatus(t.Status) {
-		maxSystemFailedTaskRetries := settings.CommitQueue.MaxSystemFailedTaskRetries
-		if maxSystemFailedTaskRetries > 0 {
-			maxExecution = maxSystemFailedTaskRetries
-		}
-	}
-	// if we've reached the max number of executions for this task, mark it as finished and failed
-	if t.Execution >= maxExecution {
-		// restarting from the UI bypasses the restart cap
-		msg := fmt.Sprintf("task '%s' reached max execution %d: ", t.Id, maxExecution)
```

Comment:
This logging doesn't seem useful, and seems like something we log when we _do_ want to do this. Unless we intend to be alerting on this behavior, I'm inclined to remove it in favor of better logging practices.